a.core_concepts.md

```diff
@@ -291,9 +327,9 @@ kubectl exec -it nginx -- /bin/sh
 <p>
 
 ```bash
-kubectl run busybox --image=busybox -it --restart=Never -- echo 'hello world'
```

Comment:
Same as before the --restart=Never is needed to actually have the pod run and output